server/go/.dockerignore

```diff
@@ -0,0 +1,2 @@
+bin/
+cover.*
```

Comment:
nit: missing extra line at the end of file

Reply:
same here... I re-cloned the repo for a sanity check.